app/bundles/LeadBundle/Form/Type/FieldType.php

```diff
@@ -244,13 +244,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                         $properties = $data->getProperties();
                     }
 
-                    $form->add(
+                $propertiesList['list'] = isset($properties['list']) && 'lookup' === $type ? array_flip(array_filter($properties['list'])) : $properties['list'];
+
+                $form->add(
```

Comment:
There is some weird indentation.
```suggestion
                    $propertiesList['list'] = isset($properties['list']) && 'lookup' === $type ? array_flip(array_filter($properties['list'])) : $properties['list'];

                    $form->add(
```